mqtt/mqtt-bridge/src/connectivity_handler.rs

```diff
@@ -0,0 +1,177 @@
+#![allow(dead_code)] // TODO remove when ready
+
+use mqtt3::Event;
+use tokio::sync::mpsc::UnboundedSender;
+use tracing::{debug, info, warn};
+
+use crate::bridge::{BridgeError, BridgeMessage, ConnectivityState};
+
+impl ToString for ConnectivityState {
```

Comment:
it's better to put trait bloks after impl. In general recommended patten is:
- struct
- impl for struct
- impl trait for struct
- free fn